tests/components/plugwise/test_init.py

```
@@ -394,8 +416,8 @@ async def test_delete_removed_device(
     """Test device removal at integration init."""
     data = mock_smile_adam_heat_cool.async_update.return_value
 
-    device_entry = device_registry.async_get_device(
-        identifiers={(DOMAIN, "14df5c4dc8cb4ba69f9d1ac0eaf7c5c6")}
+    device_entry = device_registry.async_get_device_by_identifier(
+        (DOMAIN, "14df5c4dc8cb4ba69f9d1ac0eaf7c5c6"), mock_config_entry.entry_id
```

Ruff (0.16.3)

[error] Undefined name `mock_config_entry`

(F821)